packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -14,7 +14,8 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
     error NotNominated(address addr);
 
     error InvalidCandidate(address addr);
-    error InvalidCandidatesCount();
+    error InvalidCandidatesAddresses();
```

Comment:
In both these errors, the grammar is incorrect. Usually you use a single plural in a sentence (not sure if that's the proper grammar rule 😛), so it would be:
`InvalidCandidateAddresses` and `InvalidCandidatePositions`

Reply:
Dividided the generic errors, and named them to `InvalidCandidate` on the check if its nominated, `InvalidCandidatePriority` when  priorities are not correct & `DuplicateCandidatePriority`